footerTest.js

```diff
@@ -1,4 +1,7 @@
 const puppeteer = require('puppeteer');
+const EventEmitter = require('events');
+
+EventEmitter.defaultMaxListeners = 15;
```

Comment:
Why did you change this 
is this required

Reply:
Yes, by default max listeners are only 10...so after adding new footers I'll have to add those as well.
Though it is not required it is related to this only

Reply:
It is required for tests...It checks whether on live site footer is correctly rendered or not...But since my changes are not live yet...I will have to make another PR after this one is merged.